extension/package.json

```diff
@@ -11,7 +11,7 @@
     "cover:e2e": "nyc --silent --no-clean npm run e2e",
     "unit": "nyc mocha --exit --timeout 30000 --full-trace test/unit/**/*.spec.js",
     "integration": "nyc mocha --exit --timeout 30000 --full-trace test/integration/**/*.spec.js",
-    "e2e": "nyc mocha --exit --timeout 30000 --full-trace test/e2e/**/*.spec.js",
+    "e2e": "nyc mocha --exit --timeout 60000 --full-trace test/e2e/**/*.spec.js",
```

Comment:
Just be careful there are 2 timeouts here - one is this mocha and the other is from puppeteer (https://stackoverflow.com/questions/52163547/node-js-puppeteer-how-to-set-navigation-timeout). We need to investigate a bit more which one is failing.

Reply:
Let me know what is the solution here, exception:

1) ::klarnaPayment::
       when payment method is klarna and process is done correctly, then it should successfully finish the payment:
     Error: Timeout of 30000ms exceeded. For async tests and hooks, ensure "done()" is called; if returning a Promise, ensure it resolves. (/home/travis/build/commercetools/commercetools-adyen-integration/extension/test/e2e/klarna.spec.js)
      at Test.Runnable._timeoutError (/home/travis/build/commercetools/commercetools-adyen-integration/extension/node_modules/mocha/lib/runnable.js:425:10)
      at Timeout.<anonymous> (/home/travis/build/commercetools/commercetools-adyen-integration/extension/node_modules/mocha/lib/runnable.js:237:24)
      at listOnTimeout (internal/timers.js:549:17)
      at processTimers (internal/timers.js:492:7)

Reply:
Since it's only klarna test, we can change timeout only for this test now: https://stackoverflow.com/a/23492442/1293839
I tried it locally to see the error, but it didn't fail 🤷 So for now it's hard to say why.

Reply:
Yes, but those methods are async so I don't want to use promises.

Reply:
What is your suggestion? @lojzatran

Reply:
The idea is to change `()=> {}` to `function () {}`, so you can use `this.timeout(60000)` that is in the SO link:
```
describe("something", async function () {
    this.timeout(5000); 

    // tests...
});
```

Reply:
OH yes, I forgot this thing :) .

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/330/commits/55be4d6f69d506f5a62986d2d8feeab5c1b59dad